python-main.js

```diff
@@ -915,6 +915,14 @@ function web_editor(config) {
         }
     }
 
+    function invalidFileWarning(check){
```

Comment:
Can we change `check` for `fileExtension`? It's more clear what it is that way.

Reply:
I've changed it to fileType so it will be appropriate for the makecode hex files as well. Does that work for you?

Reply:
Perfect